Microsoft.Toolkit.Uwp.UI.Controls/AdaptiveGridView/AdaptiveGridView.cs

```diff
@@ -43,6 +43,14 @@ public AdaptiveGridView()
             Items.VectorChanged += ItemsOnVectorChanged;
             Loaded += OnLoaded;
             Unloaded += OnUnloaded;
+
+            // Define ItemContainerStyle in code rather than using the DefaultStyle
+            // to avoid having to define the entire style of a GridView. This can still
```

Comment:
No. I meant GridView. If you define a DefaultStyle for the control, and only set the ItemContainerStyle, it loses all other styling. The AdaptiveGridview ended up looking like a ListView